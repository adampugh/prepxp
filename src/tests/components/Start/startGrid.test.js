import React from "react";
import { shallow } from "enzyme";

import StartGrid from "../../../components/Start/startGrid";
import user from "../../fixtures/userLists";

describe("<StartGrid />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<StartGrid questions={user.lists[0].questions}/>); 
        expect(wrapper).toMatchSnapshot();
    });
});