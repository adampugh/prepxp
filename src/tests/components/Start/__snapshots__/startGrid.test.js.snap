// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartGrid /> should match snapshot 1`] = `
<section
  className="start"
>
  <div
    className="container-s"
  >
    <div
      className="start__box"
    >
      <div
        className="start__questionBox"
      >
        <h2>
          <span>
            Q1 
          </span>
          What is the difference between let and var?
        </h2>
        <button
          className="btn"
          onClick={[Function]}
        >
          Show answer
        </button>
      </div>
      <div
        className="start__previousAnswer start__previousAnswer--black"
      >
        <p>
          Answer hidden
        </p>
      </div>
      <div
        className="start__answerBox"
      >
        <textarea
          id="answer__textarea"
          onChange={[Function]}
        />
      </div>
      <div
        className="start__button__div"
      >
        <button
          className="btn"
          onClick={[Function]}
        >
          Save
        </button>
        <button
          className="btn"
          onClick={[Function]}
        >
          Next
        </button>
      </div>
    </div>
  </div>
</section>
`;
